Vulcanizing failed while building Tensorboard

Hi,
I am getting the following error related to Vulcanizing while building Tensorboard:
**$ bazel run //tensorboard -- --logdir ./my_logs**
INFO: Analysed target //tensorboard:tensorboard (80 packages loaded).
INFO: Found 1 target...
**ERROR: C:/users/deepika.d.agarwal/desktop/what-if/tensorboard-master/tensorboard/components/BUILD:40:1: Vulcanizing /trace_viewer.html failed (Exit 1)**
Error: Could not find or load main class %1$tH:%1$tM:%1$tS.%1$tL
Caused by: java.lang.ClassNotFoundException: %1$tH:%1$tM:%1$tS.%1$tL
Target //tensorboard:tensorboard failed to build

Pls help if I am missing something here.
Thanks

--- a/tensorboard/java/org/tensorflow/tensorboard/vulcanize/BUILD
+++ b/tensorboard/java/org/tensorflow/tensorboard/vulcanize/BUILD
@@ -8,7 +8,6 @@ java_binary(
     jvm_flags = [
         "-Xss20m",  # JSCompiler needs big stacks for recursive parsing
         "-XX:+UseParallelGC",  # Best GC when app isn't latency sensitive
-        "-Djava.util.logging.SimpleFormatter.format='%1$$tY-%1$$tm-%1$$td %1$$tH:%1$$tM:%1$$tS.%1$$tL %4$$-6s %5$$s%6$$s%n'",  # Less log spam
     ],
     visibility = ["//visibility:public"],
     deps = [

--- a/tensorboard/pip_package/setup.py
+++ b/tensorboard/pip_package/setup.py
@@ -31,7 +31,7 @@ REQUIRED_PACKAGES = [
     'numpy >= 1.12.0',
     'protobuf >= 3.4.0',
     'six >= 1.10.0',
-    'werkzeug >= 0.11.10',
+    'werkzeug >= 0.11.15',
     # python3 specifically requires wheel 0.26
     'wheel; python_version < "3"',
     'wheel >= 0.26; python_version >= "3"',

--- a/tensorboard/scripts/execrooter.py
+++ b/tensorboard/scripts/execrooter.py
@@ -44,7 +44,7 @@ def run(inputs, program, outputs):
       if not os.path.exists(parent):
         os.makedirs(parent)
       # Use symlink if possible and not on Windows, since on Windows 10
-	  # symlinks exist but they require administrator privileges to use.
+      # symlinks exist but they require administrator privileges to use.
       if hasattr(os, 'symlink') and not os.name == 'nt':
         os.symlink(os.path.join(cwd, real), os.path.join(root, fake))
       else:

--- a/tensorboard/scripts/execrooter.py
+++ b/tensorboard/scripts/execrooter.py
@@ -43,14 +43,23 @@ def run(inputs, program, outputs):
       parent = os.path.join(root, os.path.dirname(fake))
       if not os.path.exists(parent):
         os.makedirs(parent)
-      os.symlink(os.path.join(cwd, real), os.path.join(root, fake))
+      # Use symlink if possible and not on Windows, since on Windows 10
+	  # symlinks exist but they require administrator privileges to use.
+      if hasattr(os, 'symlink') and not os.name == 'nt':
+        os.symlink(os.path.join(cwd, real), os.path.join(root, fake))
+      else:
+        shutil.copyfile(os.path.join(cwd, real), os.path.join(root, fake))
     if subprocess.call(program + [root]) != 0:
       return 1
     for fake, real in outputs:
       shutil.copyfile(os.path.join(root, fake), real)
     return 0
   finally:
-    shutil.rmtree(root)
+    try:
+      shutil.rmtree(root)
+    except EnvironmentError:
+      # Ignore "file in use" errors on Windows; ok since it's just a tmpdir.
+      pass
 
 
 def main(args):

